test/powershell/Host/PSVersionTable.Tests.ps1

```diff
@@ -1,4 +1,29 @@
 Describe "PSVersionTable" -Tags "CI" {
+
+    BeforeAll {
+        $powershellProcess=Get-Process -id $pid
+        $rootPath = Split-Path -Path $powershellProcess.path -Parent
+        $sma = Get-Item (Join-Path $rootPath "System.Management.Automation.dll")
```

Comment:
Why not just use `Join-Path $PSHome System.Management.Automation.dll`?

Reply:
Join-Path returns string/ We need `FileInfo` to get `VersionInfo`.

Reply:
I know, I meant using that to replace the existing `Join-Path ...` so you don't need the `get-process` and `split-path`.

Reply:
😄 Clear.